tycho-indexer/src/services/ws.rs

```diff
@@ -107,7 +107,32 @@ impl WsActor {
         stream: web::Payload,
         data: web::Data<WsData>,
     ) -> Result<HttpResponse, Error> {
-        ws::start(WsActor::new(data), &req, stream)
+        let ws_actor = WsActor::new(data);
+
+        // metrics
+        let user_agent = req
+            .headers()
+            .get("user-agent")
+            .map(|value| {
+                value
+                    .to_str()
+                    .unwrap_or_default()
+                    .to_string()
+            })
+            .unwrap_or_default();
+        let api_key = req
+            .headers()
+            .get("authorization")
```

Comment:
It's not a good practice to log API Keys - as they are personalized and sensitive. It's better to inject the username on the authentication layer and collect it here

Reply:
Okay, that's a bigger change than I was wanting to do so I will just remove the api key part entirely. We don't easily have access to the username when creating this connection request header.